app/controllers/projects_controller.rb

```diff
@@ -23,6 +28,15 @@ def new
 
   # GET /projects/1/edit
   def edit
+    setup_metric_configs(@project)
```

Comment:
Yea. I've done some tests as well. And I think this line is redundant. This function is already called when rendering _form.html. And also we no longer need to include ProjectsHelper then.